cosmos/job/JobManager.py

```diff
@@ -85,14 +80,17 @@ def run_tasks(self, tasks):
         # TODO parallelize this for speed.  Means having all ORM stuff outside Job Submission.
         map(self.submit_task, tasks, commands)
 
-    def terminate(self):
+    def terminate(self, is_cleanup=False):
```

Comment:
why is this is_cleanup stuff necessary?  it looks like you're killing tasks that aren't running

Reply:
Correct. Kubernetes does not delete a job after it completes successfully

Reply:
Kubernetes does not automatically kill jobs that have completed successfully.